docs/advanced_documentation/build-guide.md

```diff
@@ -100,40 +100,54 @@ Once you have prepared the build dependencies,
 you can install the library from source in develop mode with the development dependency. 
```

Comment:
```suggestion
you can install the library from source in development mode with the development dependency. 
```
I think the official term for this is either "development mode" or "editable installs"